code/processes/dqe.q

```diff
@@ -16,25 +16,80 @@ readdqeconfig:{[file]
 
 gethandles:{exec procname,proctype,w from .servers.SERVERS where (procname in x) | (proctype in x)};
 
-tableexists:{x in tables[]};                                                                                    /- function to check for table, param is table name as a symbol
+tableexists:{[tab]                                                                                              /- function to check for table, param is table name as a symbol
+  result:();
+  .lg.o[`dqe;"checking if ", (string tab), " table exists"];
+  $[1=a:tab in tables[];result:(1b;((string tab)," table exists"));result:(0b;(string tab)," missing from process")]
```

Comment:
`a` not used.
```suggestion
  result:$[1=a:tab in tables[];(1b;s," table exists");(0b;s," missing from process")]
```
If `result` is not a global variable then this should return directly.